modules/ROOT/pages/7.7.1-release-notes.adoc

```diff
@@ -65,6 +65,14 @@ In {productname} {release-version}, an issue was identified where skin content C
 
 This issue has been resolved in {productname} {release-version}. The bundling process now correctly incorporates CSS styles into the generated JavaScript resources, preventing truncation and ensuring consistent styling across the editor UI.
 
+=== Context forms used to disappear if their input was disabled in the `onSetup` API.
+// #TINY-11890
+
+Previously, an issue was identified where disabling a context form's input in the `onSetup` function caused the form to disappear due to focus being set on a non-focusable element.
+
+In {productname} {release-version}, this issue has been resolved by ensuring that the fallback focus is directed to a focusable element. As a result, even when the input is disabled, the focus remains intact, preventing the context form from disappearing.
```

Comment:
> the focus remains intact

I'm not entirely sure about this sentence (but I'm not a native English speaker so I could be completely wrong :sweat_smile: ), maybe I would say that the focus now is moved on a valid element, meaning a focusable element, saying that the focus remains intact sounds to me like it doesn't move at all, but I repeat I'm not a native English speaker so I could misunderstand the nuance of this sentence.

For the rest good job :+1:

Reply:
Just a suggestion, if we want to update this before we ship it...
```suggestion
In {productname} {release-version}, this issue has been resolved by ensuring that the fallback focus is directed to a focusable element. As a result, even when the input is disabled, the focus is properly redirected, preventing the context form from disappearing.
```